### PR TITLE
DSD-1336: Updating aria labels for Tabs mobile next/prev buttons

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Fixes JIRA ticket [DSD-1336](https://jira.nypl.org/browse/DSD-1336)

## This PR does the following:

- Updates the `aria-label` for the previous/next buttons to "Scroll tabs left" and "Scroll tabs right", respectively, for better description.

## How has this been tested?

Locally.

## Accessibility concerns or updates

- Better descriptive text for the previous/next button.

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] Review the Vercel preview deployment once it is ready.

### CODE REVIEW

File: src/components/Tabs/Tabs.tsx [hunk 1]
```diff
@@ -186,7 +186,7 @@ export const Tabs = chakra(
 
       const previousButton = (
         <Button
-          aria-label="Previous"
+          aria-label="Scroll tabs left"
```
Comment (bigfishdesign13) [inline src/components/Tabs/Tabs.tsx:189]: Looking at this update, I feel we should also add a more descriptive `title` prop on the `Icon` components.  Can you also add `title="left arrow"` and `title="right arrow"` to the respective `Icon` components?

**Review by bigfishdesign13 — CHANGES_REQUESTED** [review]
One additional change.

File: src/components/Tabs/Tabs.tsx [hunk 2]
```diff
@@ -199,6 +199,7 @@ export const Tabs = chakra(
             id={`tabs-previous-icon-${id}`}
             name="arrow"
             size="small"
+            title="left arrow"
```
Comment (bigfishdesign13) [inline src/components/Tabs/Tabs.tsx]: I originally asked for the Icons to be titled "left arrow" and "right arrow", but I think what you accidentally did for the Next button works better.  With that, change this to "previous arrow."

**Review by bigfishdesign13 — CHANGES_REQUESTED** [review]
One more small change.

**Review by bigfishdesign13 — APPROVED** [review]
The labeling updates look good, but did the updates in `ci.yml` do whay was expected?

**Comment by EdwinGuzman** [comment]
> The labeling updates look good, but did the updates in ci.yml do whay was expected?

Yes? Sort of.

**Pull request merged**